src/Text/Pandoc/App.hs

```diff
@@ -218,9 +218,13 @@ convertWithOpts opts = do
     metadataFromFile <-
       case optMetadataFiles opts of
         []    -> return mempty
-        paths -> mconcat <$>
-           mapM (\path -> do raw <- readFileStrict path
-                             yamlToMeta readerOpts (Just path) raw) paths
+        paths -> do
+          mconcat <$>
+            mapM
+              (\path -> do
+                raw <- readMetadataFile path
+                yamlToMeta readerOpts (Just path) raw)
+              paths
```

Comment:
Here it would be helpful if your change were the minimal one necessary: just changing `readFileStrict` to `readMetadataFile` and leaving everything else the same.  With the reformatting, it becomes more difficult to see what has changed.  (Also, there's now a redundant `do`.)

Reply:
Absolutely, sorry about that. Changed in fixup.